docs/src/content/docs/guides/onetime-remittance-fixed-debit.mdx

```diff
@@ -0,0 +1,508 @@
+        "actions": ["create", "read"]
+      }
+    ]
+  },
+  "continue": {
+    "access_token": {
+      "value": "..." // access token for continuing the request
+    },
+    "uri": "https://auth.cloudninebank.example.com/continue/{...}" // continuation request uri
+  }
+}
+```
+
+</details>
+
+### 5. Request the creation of a quote resource
+
+Use the access token received in the previous step to call the <Badge text="POST" variant="success" /> [Create Quote API](/apis/resource-server/operations/create-quote).
+
+This call requests that a quote resource be created on the sender's wallet account. The request must contain the `receiver`, which is the recipient's incoming payment `id`, along with the `debitAmount`, which is the exact amount the sender wants to pay.
+
+The `debitAmount` specifies that the sender will pay exactly $100 USD, and the recipient will receive whatever amount remains after currency conversion.
+
+<Tabs syncKey="lang">
+<TabItem label='TypeScript/JavaScript' icon='seti:javascript'>
+  ```ts wrap
+  const senderQuote = await client.quote.create(
+    {
+      url: senderWalletAddress.resourceServer,
+      accessToken: senderQuoteGrant.access_token.value
+    },
+    {
+      method: 'ilp',
+      walletAddress: senderWalletAddress.id,
+      receiver: recipientIncomingPayment.id,
+      debitAmount: {
+        value: '10000', 
+        assetCode: 'USD',
+        assetScale: 2
+      }
+    }
+  )
+```
+</TabItem>
+</Tabs>
+
+The response returns a `receiveAmount`, a `debitAmount`, and other required information.
+
+- `debitAmount` - The amount the sender must pay (exactly $100.00 USD in our example).
+- `receiveAmount` - The amount the recipient will actually receive ($1,700.00 MXN in our example) after currency conversion.
+
+<details>
+<summary>Example response</summary>
+  The following shows an example response from the sender's wallet provider.
+
+```json wrap
+{
+  "id": "https://cloudninebank.example.com/quotes/{...}", // url identifying the quote
+  "walletAddress": "https://cloudninebank.example.com/sender",
+  "receiver": "https://happylifebank.example.com/incoming-payments/{...}", // url of the incoming payment the quote is created for
+  "debitAmount": {
+    "value": "10000", // Sender pays exactly $100.00 USD
+    "assetCode": "USD",
+    "assetScale": 2
+  },
+  "receiveAmount": {
+    "value": "170000", // Recipient receives $1,700.00 MXN  after currency conversion
+    "assetCode": "MXN",
+    "assetScale": 2
+  },
+  "method": "ilp",
+  "createdAt": "2025-03-12T23:22:51.50Z"
+}
+```
+
+</details>
+
+### 6. Request an interactive outgoing payment grant
+
+Use the sender's `authServer` information received in Step 1 to call the <Badge text="POST" variant="success" /> [Grant Request API](/apis/auth-server/operations/post-request).
+
+This call obtains an access token that allows your app to request that an outgoing payment resource be created on the sender's wallet account.
+
+<InteractionReq />
```

Comment:
Remove and replace w/ manually entered text. Change the reference from "resource owner" to whatever is most appropriate for this guide (e.g., customer, sender, your user, etc)